examples/chat/src/ui.tsx

```diff
@@ -31,7 +31,7 @@ export function BubbleContainer(props: {
   fromMe: boolean | undefined;
 }) {
   const align = props.fromMe ? "items-end" : "items-start";
-  return <div className={`${align} flex flex-col m-2`}>{props.children}</div>;
+  return <div className={`${align} flex flex-col m-2`} role="row">{props.children}</div>;
```

Comment:
Added some basic roles to match the a11y patterns implemented on Whatsapp web.

Initially meant to simplify testing but I've preferred to use text selectors in the end.